Content.Shared/ReagentOnItem/Components/SpaceGlueOnItemComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.ReagentOnItem;
+
+[RegisterComponent]
+public sealed partial class SpaceGlueOnItemComponent : ReagentOnItemComponent
+{
+    [DataField("timeOfNextCheck", customTypeSerializer: typeof(TimeOffsetSerializer)), ViewVariables(VVAccess.ReadWrite)]
+    public TimeSpan TimeOfNextCheck;
+
```

Comment:
This isn't handling pausing just use the sourcegen.

Reply:
OK Tayrtahn helped me fix it! Also just to be clear I did realize that didn't handle pausing I just thought it was a more understandable way of doing it than the original one.